db/migration/202009171251-init-database.go

```diff
@@ -77,6 +77,9 @@ CREATE TABLE IF NOT EXISTS workflow_worker_map (
         , worker_id UUID NOT NULL
 );
 
+ALTER TABLE workflow_worker_map DROP CONSTRAINT IF EXISTS workflow_worker_pkey;
+ALTER TABLE workflow_worker_map ADD CONSTRAINT workflow_worker_pkey PRIMARY KEY (workflow_id, worker_id);
```

Comment:
This needs to be done in a different migration file/procedure so that it will be run automatically.

Would it be better to do a concurrent unique index and then attach as a table constraint? Migrations will most likely be run in parallel with existing running code and locking the table doesn't sound like a good idea.

Reply:
Done. Actually a unique index is sufficient in this case. I moved it to a separate migration 202010221010-add-unique-index.